src/scripts/slate/utils.js

```diff
@@ -63,4 +63,19 @@ slate.utils = {
     }
     return result;
   }
+
+  /**
+   * _.defaultTo from lodash
+   * Checks `value` to determine whether a default value should be returned in
+   * its place. The `defaultValue` is returned if `value` is `NaN`, `null`,
+   * or `undefined`.
+   * Source: https://github.com/lodash/lodash/blob/master/defaultTo.js
+   *
+   * @param {*} value - Value to check
+   * @param {*} defaultValue - Default value
+   * @returns {*} - Returns the resolved value
+   */
+  function defaultTo(value, defaultValue) {
```

Comment:
This will need to be defined as a property of `utils`.

```js
defaultTo: function(value, defaultValue) {
  return (value == null || value !== value) ? defaultValue : value
}
```